tests/Builder/Accumulator/FirstAccumulatorTest.php

```diff
@@ -0,0 +1,71 @@
+<?php
+
+declare(strict_types=1);
+
+namespace MongoDB\Tests\Builder\Accumulator;
+
+use MongoDB\Builder\Accumulator;
+use MongoDB\Builder\Expression;
+use MongoDB\Builder\Pipeline;
+use MongoDB\Builder\Stage;
+use MongoDB\Tests\Builder\PipelineTestCase;
+
+use function MongoDB\object;
+
+/**
+ * Test $first accumulator
+ */
+class FirstAccumulatorTest extends PipelineTestCase
+{
+    public function testMissingData(): void
+    {
+        $pipeline = new Pipeline(
+            Stage::sort(object(
+                item: 1, // @todo accept true
```

Comment:
Is this TODO tracked anywhere? [PHPORM-105](https://jira.mongodb.org/browse/PHPORM-105) is the only sort-related ticket I found.

Reply:
I don't have created an issue for all operators, we will have to check the behavior of all operators when implementing tests.